tests/unit/data-access/MwApiLexemeCreator.test.ts

```diff
@@ -12,6 +12,10 @@ function mockRejectedDeferred( ...args: unknown[] ): unknown {
 	};
 }
 
+function mwGetUrl( pageName: string | null ) {
+	return `https://wiki.org/${pageName}`;
```

Comment:
Let’s use one of the standard example domains, e.g. `example.com` or `wiki.example`.